app/src/main/res/values/strings.xml

```diff
@@ -59,4 +59,7 @@
     <string name="check_icon_contentdescription">Selektier Icon</string>
     <string name="chevron_icon_contentdescription">Navigations Icon</string>
     <string name="disposal_image_contentdescription">Abfallgruppe %s</string>
+    <string name="bottom_navigation_calendar_title">Kalender</string>
+    <string name="bottom_navigation_information_title">Information</string>
+    <string name="bottom_navigation_settings_title">Einstellungen</string>
```

Comment:
I would propose to have a specific prefix for all content_description-keys on locolise.biz -> this would prevent translators to search for strings in the app, which are not visible. What do you think, @kaiwidmer ?

Reply:
@tschuls I've added the suffix "contentdescription" for these three keys (like the other ones)